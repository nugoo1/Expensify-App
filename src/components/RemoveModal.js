import React from 'react';
import Modal from 'react-modal';

const RemoveModal = (props) => (
        <Modal
            isOpen={!!props.selectedOption}
            onRequestClose={props.onCancelHandler}
            contentLabel="Confirm Remove Expense"
            closeTimeoutMS={200}
            className="modal"
        >
            <h3 className="modal__text">Are you sure you want to remove this expense?</h3>
            <button className="button modal__button" onClick={props.confirmRemoveExpense}>Remove</button>
            <button className="button button--secondary" onClick={props.onCancelHandler}>Cancel</button>

        </Modal>
    );


export default RemoveModal;
