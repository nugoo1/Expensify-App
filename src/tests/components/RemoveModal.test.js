import React from 'react';
import { shallow } from 'enzyme';
import expenses from '../fixtures/expenses';
import RemoveModal from '../../components/RemoveModal';

test('should render RemoveModal page', () => {
    const wrapper = shallow(<RemoveModal />)
    expect(wrapper).toMatchSnapshot();
});
