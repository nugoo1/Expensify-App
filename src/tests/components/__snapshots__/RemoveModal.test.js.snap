// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render RemoveModal page 1`] = `
<Modal
  ariaHideApp={true}
  bodyOpenClassName="ReactModal__Body--open"
  className="modal"
  closeTimeoutMS={200}
  contentLabel="Confirm Remove Expense"
  isOpen={false}
  parentSelector={[Function]}
  portalClassName="ReactModalPortal"
  shouldCloseOnOverlayClick={true}
>
  <h3
    className="modal__text"
  >
    Are you sure you want to remove this expense?
  </h3>
  <button
    className="button modal__button"
  >
    Remove
  </button>
  <button
    className="button button--secondary"
  >
    Cancel
  </button>
</Modal>
`;
